src/BlazorAdmin/Services/AuthService.cs

```diff
@@ -16,12 +14,10 @@ public class AuthService
         private readonly HttpClient _httpClient;
         private readonly ILocalStorageService _localStorage;
         private readonly IJSRuntime _jSRuntime;
+        private static bool InDocker { get; set; }
```

Comment:
We shouldn't need the code for knowing if the app is running in docker in more than one place.

Reply:
InDocker in Blazor: get the result from cookie to choose the web ip and api ip.
InDocker in Api: get the ip for web.
InDocker in Web: get the ip for web and api.